UPGRADING.md

```diff
@@ -29,6 +29,7 @@ const knexInstance: Knex = knex(config)
 * v8 flags are no longer supported in cli. To pass these flags use [`NODE_OPTIONS` environment variable](https://nodejs.org/api/cli.html#cli_node_options_options).
   For example `NODE_OPTIONS="--max-old-space-size=1536" npm run knex`
 
+* Clients are now classes instead of new-able functions. Please migrate your custom clients to classes.
```

Comment:
Can you add an example of how the old code looked like and how the new one would look?

Reply:
done